Modules/Loadable/Segmentations/Widgets/qMRMLSegmentEditorWidget.cxx

```diff
@@ -2886,7 +2890,7 @@ void qMRMLSegmentEditorWidget::processEvents(vtkObject* caller, unsigned long ei
       callbackCommand->SetAbortFlag(1);
     }
   }
-  else if (callerViewNode)
+  else if (vtkMRMLAbstractViewNode* callerViewNode = vtkMRMLAbstractViewNode::SafeDownCast(caller); callerViewNode != nullptr)
```

Comment:
Readable code is more important than conciseness. I think this is also a practice we should not propagate.

Reply:
Motivation was to avoid using `callerViewNode` in the "wrong" code branch (which I originally did), this construct conveniently avoided this.

That said, I understand the readability argument and reverted back.